docs/platform/what-is-tizen/versions/tizen-7-0-m2.md

```diff
@@ -0,0 +1,112 @@
+# Tizen 7.0
+
+The second milestone (M2) release of Tizen 7.0 was announced in October 2022. 
+Tizen is an open-source operating system (OS) maintained by Samsung, and it is being developed and optimized for smart devices since 2012.
+
+The main progress of Tizen 7.0 includes work that has been done to accommodate new Tizen devices and strengthening intelligent device and service platform features. Core platform technologies are needed to prepare for our future with the Metaverse and robots.
+These new enhancements include 3D rendering engine enhancement, real-time enabled kernel, and more.
+
+## Natural user interface
+
+- **Mini Avatar UI Assistant(Consistent Interaction Through Voice)**:
+
+  Tizen provides UI assistance technology that allows users to control the UI through voice without additional processing in applications
+
+  The Mini Avatar is displayed in a separate layer, and it enables more natural interaction by showing the appropriate feedback for the situation.
+  It also shows automatically generated commands for all controllable UI objects, and when the user says a command, it sends events to a selected object.
+
+  **Figure: Mini Avatar UI Assistant**
+
+  ![MiniAvartarUIAssistance](./media/7.0_MiniAvartarUIAssistance.png)
+
+- **3D User Interface**:
+
+  Tizen provides a unified rendering engine. All UI elements have depth information and they are located in a 3D space.
+
+  As all UI elements are rendered together, we can optimize the rendering pass and minimize memory usage.
+
+	The biggest advantage is that it reduces the complexity quite significantly for creating a 3D UI.
+	This is because almost all the features used in 2D UI applications such as event handling, text input, animations, and accessibility, etc., are still available in 3D
```

Comment:
```suggestion
	This is because almost all the features used in 2D UI applications such as event handling, text input, animations, accessibility, etc., are still available in 3D
```